test/README.md

```diff
@@ -108,12 +108,20 @@ tests will fail. If this happens, remove the cache directory (and make
 sure bitcoind processes are stopped as above):
 
 ```bash
-rm -rf cache
+rm -rf test/cache
 killall bitcoind
 ```
 
 ##### Test logging
 
+The test framework logs to two files: 
+ * `<test data directory>/test_framework.log` 
+ * `<test data directory>/node<number>/regtest/debug.log`. 
+ 
+The `<test data directory>` path is by default set to the temporary directory of
+your OS, and its full path is written as the very first line of the test output.
+The `node<number>` part identifies the relevant test node (from `node1` to`node4`).
```

Comment:
Add a space after "to" here: to\`node4\`

Reply:
> ACK, but I agree that both files should be fixed up. I think 12 lines to review is small enough to include more changes.

@MarcoFalke Fair enough. I'll look into it!

Reply:
@jonatack Thanks, fixed! (blush)